### PR TITLE
Draft updates to Code of Conduct for guidelines

### PR DESCRIPTION
**Pull request by mattstratton**
Some small updates to the Code of Conduct (scoped to community guidelines)

### CODE REVIEW

File: CODE-OF-CONDUCT.md [hunk 1]
```diff
@@ -30,6 +31,12 @@ Examples of unacceptable behavior by participants include:
   address, without explicit permission
 * Other conduct which could reasonably be considered inappropriate in a
   professional setting
+  
+## Our Community Guidelines
+* Be clear and stay on topic. Communicating with strangers on the Internet can make it hard to convey or read tone, and sarcasm is frequently misunderstood. Try to use clear language, and think about how the other person will receive it.
+* Don’t cross-post the same thing in multiple GitHub Discussion topics or multiple Slack channels.
```
Comment (jaxxstorm) [inline CODE-OF-CONDUCT.md:37]: Maybe add something like "this can make it difficult for people answering your questions and creates scrollback spam"

**Review by jaxxstorm — APPROVED** [review]

File: CODE-OF-CONDUCT.md [hunk 2]
```diff
@@ -30,6 +31,12 @@ Examples of unacceptable behavior by participants include:
...
+* Public discussion is preferred to private. Avoid using Slack DMs for questions, and instead share them in public Slack channels or GitHub Discussion threads. This allows a larger audience to both share their knowledge as well as learn from your question or issue.
```
Comment (pierskarsenbarg) [inline CODE-OF-CONDUCT.md:38]: Should also add something along the lines of "if you're having a problem then chances are someone else is having the same problem"

File: CODE-OF-CONDUCT.md (hunk 1 above)
Comment (mattstratton) [reply CODE-OF-CONDUCT.md]: added!

File: CODE-OF-CONDUCT.md (hunk 2 above)
Comment (mattstratton) [reply CODE-OF-CONDUCT.md]: Great suggestion! I added it.

**Pull request merged**